lib/glific/flows/flow_context.ex

```diff
@@ -416,7 +416,7 @@ defmodule Glific.Flows.FlowContext do
         do: query |> where([fc], fc.id == ^parent_id),
         else: query
 
-    Repo.one(query) |> Repo.preload(:contact)
+    Repo.one(query) |> Repo.preload([:contact, :flow])
```

Comment:
this should be

query |> preload([:contact, :flow])) |> Repo.one()

Reply:
@dlobo 
Need to talk about this.
As per my understanding, it will load all the context and then flow instead of the first one.

Let me know If I am missing something here.

query |> preload([:contact, :flow])) |> Repo.one()